Source/Dafny/Triggers/QuantifiersCollection.cs

```diff
@@ -285,10 +285,18 @@ internal class QuantifierGroup {
       var msg = new StringBuilder();
       var indent = addHeader ? "  " : "";
       bool suppressWarnings = Attributes.Contains(q.quantifier.Attributes, "nowarn");
+      var reportingToken = q.quantifier.tok;
+
+      void SetRelatedLocationMessage(string msg) {
+        if (reportingToken is NestedToken nestedToken) {
+          reportingToken = new NestedToken(nestedToken.Outer, nestedToken.Inner, msg);
+        }
+      }
 
       if (!TriggerUtils.NeedsAutoTriggers(q.quantifier)) { // NOTE: split and autotriggers attributes are passed down to Boogie
         var extraMsg = TriggerUtils.WantsAutoTriggers(q.quantifier) ? "" : " Note that {:autotriggers false} can cause instabilities. Consider using {:nowarn}, {:matchingloop} (not great either), or a manual trigger instead.";
         msg.AppendFormat("Not generating triggers for \"{0}\".{1}", Printer.ExprToString(q.quantifier.Term), extraMsg).AppendLine();
+        SetRelatedLocationMessage("Trigger position:");
```

Comment:
Could we have a newline before this?  Otherwise currently the message looks like this:

```
constructors-cause-matching-loops.dfy(10,9): Info: Selected triggers: {Succ(s)}
 Rejected triggers:
   {f(s)} (may loop with "f(Succ(s))")
   {f(Succ(s))} (more specific than {Succ(s)}) Trigger position: constructors-cause-matching-loops.dfy(10,40)

Dafny program verifier finished with 1 verified, 0 errors
```

This is confusing: it's not clear *which* trigger the location points to.

Reply:
I changed:

` Trigger position`

to newline  +
` Selected trigger position`

Reply:
Sounds good.

Reply:
Except wait, this line is for the case where no triggers are being generated because the user turned that feature off, so that data looks wrong too.